app/views/posts/_feed.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="col-12 col-md-6 mb-4">
+  
+
+  <% if user_signed_in? %>
```

Comment:
OPTIONAL

- [ ] It is a good practice to have zero logic in the views directory. Create methods in the helpers directory and call the methods in the views directory instead.
Read this [article](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4) for a head-start. Another great piece to read is this [article](https://brandonhilkert.com/blog/refactoring-logic-from-a-rails-view/)